ci/travis/before_install-linux.sh

```diff
@@ -59,3 +59,5 @@ pip install --use-wheel --user -r bindings/py/requirements.txt --quiet || exit
 
 pip install --user pycapnp==0.5.8 || exit
 pip install cpp-coveralls --user
+
+export PIP_USER=--user
```

Comment:
See my other comment, but if using `pip`, this line need only be `PIP_USER=1` per `pip`'s ability to translate env vars to cli options (https://pip.pypa.io/en/stable/user_guide/#environment-variables).